nautobot/extras/models/relationships.py

```diff
@@ -366,7 +366,7 @@ def clean(self):
                 )
 
 
-class RelationshipAssociation(BaseModel):
+class RelationshipAssociation(BaseModel, ChangeLoggedModel):
```

Comment:
this is done to keep the `ViewTestCases.DeleteObjectViewTestCase` because this test is now validating for ObjectChange creation when deleting an item